### ISSUE TITLE
chips example has a keyboard trap in Firefox

### ISSUE THREAD

**Issue by mfairchild365** [angular/components#15011]
#### What is the expected behavior?
Users must be able to use the tab key on their keyboard to access to every element in the tab order on the page.

#### What is the current behavior?
Users are unable to tab past the "Chips Autocomplete" example when using Firefox.

#### What are the steps to reproduce?
1. Open Firefox 64
2. Open the [chips examples page](https://material.angular.io/components/chips/examples)
3. Press tab until you reach the chips in the "Chips with input" example (located under the "Chips Autocomplete" example)
4. Observe that it is not possible to reach the chips in the "Chips with input" example because focus is trapped to the "Chips Autocomplete" example.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- FireFox 64

#### Is there anything else we should know?
Also note this works fine in some other browsers such as Google Chrome.
